### ISSUE TITLE
[Task]: Collect data on breakup of time taken by action execution on the backend

### ISSUE THREAD

**Issue by SatishGandham** [appsmithorg/appsmith#12644]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

One of the things we can optimise to improve the first load time and applications performance in general is the execute calls. Some are taking upto 2seconds.

Before we start optimising them, we will need a breakup of the time taken between

- Time spent in making the api call, reading from db.
- Serialising/processing the response.

**Comment by nidhi-nair** [comment]
Closing in favour of #17752
